config/production.json

```diff
@@ -69,6 +69,7 @@
 		"plans/personal-plan": true,
 		"post-editor/author-selector": true,
 		"post-editor/insert-menu": true,
+		"post-editor/track-paste-events": true,
```

Comment:
Fair enough. Seems like you could add to the _shared config file and disable on desktop if needed: https://github.com/Automattic/wp-calypso/blob/master/config/_shared.json